plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/TestMySqlTypeMapping.java

```diff
@@ -599,13 +599,13 @@ private void testDate(ZoneId sessionZone)
     @Test
     public void testTimeFromMySql()
     {
-        testTimestampFromMySql(UTC);
-        testTimestampFromMySql(ZoneId.systemDefault());
+        testTimeFromMySql(UTC);
+        testTimeFromMySql(ZoneId.systemDefault());
         // no DST in 1970, but has DST in later years (e.g. 2018)
-        testTimestampFromMySql(ZoneId.of("Europe/Vilnius"));
+        testTimeFromMySql(ZoneId.of("Europe/Vilnius"));
         // minutes offset change since 1970-01-01, no DST
-        testTimestampFromMySql(ZoneId.of("Asia/Kathmandu"));
-        testTimestampFromMySql(TestingSession.DEFAULT_TIME_ZONE_KEY.getZoneId());
+        testTimeFromMySql(ZoneId.of("Asia/Kathmandu"));
+        testTimeFromMySql(TestingSession.DEFAULT_TIME_ZONE_KEY.getZoneId());
```

Comment:
Relates to https://github.com/trinodb/trino/commit/b561a3399aef3b9445da31812d2dcda75189bbf6